src/RevTracer.cc

```diff
@@ -0,0 +1,336 @@
+//
+// _RevTracer_cc_
+//
+// Copyright (C) 2017-2023 Tactical Computing Laboratories, LLC
+// All Rights Reserved
+// contact@tactcomplabs.com
+//
+// See LICENSE in the top level directory for licensing details
+//
+//
+
+#include <sstream>
+#include <iomanip>
+#include <string>
+
+#include "../include/RevTracer.h"
+#include "RevTracer.h"
+
+using namespace SST::RevCPU;
+
+RevTracer::RevTracer(std::string Name, SST::Output *o)
+    : name(Name), pOutput(o), outputEnabled(false), lastPC(0), insn(0), 
```

Comment:
Optimization tip:  `name(std::move(Name))` makes it a shallow instead of deep copy.

Also, constructor parameter names don't have to be different from the member names they initialize. `name(std::move(name))` is perfectly fine; the initialized member `name` is in class scope; the initializer expression `std::move(name)`'s `name` is in block (function) scope.